config/locales/es-MX.yml

```diff
@@ -84,7 +84,7 @@ es-MX:
         new_password: Contraseña nueva
       new:
         forgot_your_password: '¿Ha olvidado su contraseña?'
-        send_me_reset_password_instructions: Enviarme instrucciones para restablecer la contraseña
+        send_me_reset_password_instructions: 'Restablecer la contraseña'
```

Comment:
Just an opinion, you can either choose to address it or let it pass. This can be reduced to `Restablecer contraseña`. LMK if this is ok or you can keep it like it is. Up to you.